### DISCUSSION TITLE
My css file and the js file is not being applied on the github page

### DISCUSSION THREAD

**Discussion by gengamask** [community/community#49042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi.

So I have this simple codebase for the scheduler and the if I run this on VS code live server, everything is working great, but when I deploy it on the github page, it all disappears. bootstraps is being applied, but the dayjs, script.js, and style.css are not. 

repo: https://github.com/gengamask/scheduler

Live demo: https://gengamask.github.io/scheduler/

**Comment by Anmol-Baranwal** [comment]
Show, what it should have been if it worked properly.

Anyway, it works fine as you can see [here](https://github.com/Anmol-Baranwal/Portfolio-Akanksha)

**Reply by Anmol-Baranwal** [reply]
@gengamask 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
